sqlglot/dialects/hive.py

```diff
@@ -148,6 +148,18 @@ def _to_date_sql(self: Hive.Generator, expression: exp.TsOrDsToDate) -> str:
     return f"TO_DATE({this})"
 
 
+def clusteredcolumnconstraint_sql(
+    self: Hive.Generator,
+    expression: exp.ClusteredColumnConstraint | exp.NonClusteredColumnConstraint,
+) -> str:
+    expression = expression.copy()
```

Comment:
I just put this `copy` here to avoid complicating the code, i.e. checking if there's at least one `desc` arg set to `False` etc... Shouldn't be too costly, I think.

Reply:
it needs to be done on all constraints, probably not clustered/nonclustered, those are tsql specific

Reply:
What other constraint has an `Ordered` child?

Reply:
i don't know